src/styles/components/_footer.scss

```diff
@@ -9,95 +9,74 @@ $breakpoint-footer-margin: 25em;
 
   // Content wrapper
   &__content {
-    min-height: 20.75rem; /* 332px/16 */
     display: flex;
     flex-direction: column;
-    justify-content: space-between;
-    align-items: center;
-    gap: 3.75rem; // 60px/16
+    gap: 2.25rem; // 60px/16
     text-align: center;
-    padding: 3.75rem 0; /* top&bottom: 60px/16; left&right: 0 */
-    font-size: map.get($font-sizes, "sm");
-    margin: auto;
-    line-height: 1.5rem; /* 24px/16 */
-    transition: 500ms color;
+    padding: 1.5rem 0; /* top&bottom: 60px/16px; left&right: 0 */
+    font-size: map.get($font-sizes, "xs");
     letter-spacing: 0.02em; // 2% in Figma
-
-    @media (min-width: $breakpoint-large-screen-860) {
-      line-height: 1.0625rem; /* 17px/16 */
-      font-size: map.get($font-sizes, "sm");
-    }
+    color: var(--foreground-color);
   }
 
   // Flex items of content wrapper
   &__item {
-    color: var(--foreground-color);
     width: 100%;
 
-    // the first line of the footer has a smaller margin-bottom than the rest of the footer items.
-    // this compensates for that
+    // first line of text needs extra space below it.
     &:first-child {
-      margin-bottom: -1.875rem; // 30px/16
-
-      @media (min-width: $breakpoint-large-screen-860) {
-        margin-bottom: -43px; // this is to compensate for the 17px line-height once at tablet size
-      }
+      margin-bottom: -1.5rem; // 30px/16
     }
-  }
 
-  // text-links
-  &__item > a {
```

Comment:
I added an extra modifier class, `.default-footer__content--text`, since using the direct child selector `<` with `<a>` was very prone to errors while refactoring. It was also just kind of confusing, honestly!